docs/js/doc-page.js

```diff
@@ -50,6 +138,36 @@ export default ({ data, location }) => {
     `/${rest.join('/')}/`,
   )}`;
 
+  // For Schema purposes, we want to make the relationship between pages and their parents explicit
+  // So we flatten all links (see flattenLinks above)
+  const flatLinks = flattenLinks(docLinks);
+  // Find the current expanded link
+  const expandedLink = flatLinks.find(
+    (l) => getPathFromSlug(l.link) === getPathFromSlug(slug),
+  );
+  let breadcrumbs = [];
+  let section;
+  // And check to see if it has any parent
+  if (expandedLink && expandedLink.parent) {
```

Comment:
Pages can be arbitrarily deep, right? Shouldn't this be a loop? Instead of just hard-coding 2 iterations?

Reply:
Done!